python/chronos/test/bigdl/chronos/forecaster/tf/test_mtnet_forecaster.py

```diff
@@ -61,28 +61,28 @@ def test_forecast_mtnet(self):
         # TODO hacking to fix a bug
         train_data, test_data = create_data()
         self.x_train, y_train = train_data.to_numpy()
-        self.y_train = y_train[:, :, 0]
+        self.y_train = y_train.reshape(y_train.shape[0], y_train.shape[-1])
```

Comment:
I think you are right, but the original code seems to fetch the last dim rather than the last step. [Just mention it here for double check]